content/applications/inventory_and_mrp/maintenance/maintenance_reporting.rst

```diff
@@ -0,0 +1,295 @@
+- :guilabel:`Request Date`: the date on which the maintenance request was created.
+- :guilabel:`Employee`: the employee who originally created the maintenance request.
+- :guilabel:`Technician`: the technician responsible for the maintenance request.
+- :guilabel:`Category`: the category the equipment being repaired belongs to.
+- :guilabel:`Stage`: the stage of the maintenance process the request is currently in.
+- :guilabel:`Company`: if in a multi-company environment, the company in the database the request is
+  assigned to.
+
+Calendar view
+-------------
+
+Click the :icon:`fa-calendar` :guilabel:`(calendar)` icon to change to calendar view.
+
+There are a number of options in this view type for sorting and grouping information about
+maintenance requests.
+
+In the top-left corner of the page, there is a drop-down menu set to :guilabel:`Week`, by default.
+Clicking that drop-down menu reveals the different periods of time, in which the calendar can be
+viewed: :guilabel:`Day`, :guilabel:`Month`, and :guilabel:`Year`. There is also an option to
+:guilabel:`Show weekends`, selected by default. If unselected, weekends are not shown on the
+calendar.
+
+.. image:: maintenance_reporting/maintenance-reporting-period-options.png
+   :align: center
+   :alt: Calendar period drop-down menu options.
+
+To the left of this menu, there is a :icon:`fa-arrow-left` :guilabel:`(left arrow)` icon and a
+:icon:`fa-arrow-right` :guilabel:`(right arrow)` icon. Clicking these arrows moves the calendar
+backward or forward in time, respectively.
+
+To the right of the drop-down menu set to :guilabel:`Week`, by default, is a :guilabel:`Today`
+button. Clicking this button resets the calendar to view today's date, no matter which point in time
+is being viewed before clicking it.
+
+At the far-right side of the page is a sidebar column, containing a minimized calendar set to
+today's date, and a :guilabel:`Technician` list, displaying all the *Technicians* with requests
+currently open. Click the :icon:`oi-panel-right` :guilabel:`(panel)` icon at the top of this sidebar
+to open or close the sidebar.
+
+.. note::
+   The :guilabel:`Technician` list only displays if technicians are assigned to open requests, and
+   individual technicians are only listed, if they are listed as :guilabel:`Responsible` on at least
+   **one** maintenance request form.
+
+Activity view
+-------------
+
+With the :guilabel:`Activity` view selected, all open maintenance requests are listed in their own
+row, with the ability to schedule activities related to those requests.
+
+Maintenance requests are listed in the :guilabel:`Maintenance Request` column as activities.
+Clicking a request opens a :guilabel:`Maintenance Request` popover that indicates the status of the
+request, and the responsible technician. To schedule an activity directly from the popover, click
+:icon:`fa-plus` :guilabel:`(Schedule an activity)`. This opens a :guilabel:`Schedule Activity`
```

Comment:
General comment- because there ar eno acitvities in my runbot, all I see is a +Schedule Activity button beneath the list view - when I click it, it first takes me to this:
![image](https://github.com/odoo/documentation/assets/121518652/3a32bcb5-ee16-458b-92bf-0e8139402bcd)

Reply:
That's a good point, I see what you're saying. I added in a note admonition to explain what to do when nothing is listed there, so that info is there too. Suggestion implemented ✅